templates/events.html

```diff
@@ -59,9 +59,130 @@ <h4>Recent Notifications</h4>
                 </td>
               </tr>
+              <tr>
+                <td>Proximity Sensor</td>
+                <td>Motion Detected</td>
+                <td>
+                    10:42 am
+                    27/02/2022
+                </td>
+                <td>
+                    <button type="button" class="btn btn-primary">Go to Cameras</button>
+                </td>
+              </tr>
+              <tr>
+                <td>Camera #1</td>
+                <td>Image Captured</td>
+                <td>
+                    10:41 am
+                    27/02/2022
+                </td>
+                <td>
+                    <button type="button" class="btn btn-primary">Go to Train</button>
+                </td>
+              </tr>
+              <tr>
+                <td>Camera #2</td>
+                <td>Face Detected</td>
+                <td>
+                    10:40 am
+                    27/02/2022
+                </td>
+                <td>
+                    <button type="button" class="btn btn-primary">Go to Train</button>
+                </td>
+              </tr>
+              <tr>
+                <td>Camera #1</td>
+                <td>Fault Detected</td>
+                <td>
+                    6:42 am
+                    27/02/2022
+                </td>
+                <td>
+                    <button type="button" class="btn btn-primary">Go to Train</button>
+                </td>
+              </tr>
+              <tr>
+                <td>Proximity Sensor</td>
+                <td>Motion Detected</td>
+                <td>
+                    10:42 am
+                    27/02/2022
+                </td>
+                <td>
+                    <button type="button" class="btn btn-primary">Go to Cameras</button>
+                </td>
+              </tr>
+              <tr>
+                <td>Camera #1</td>
+                <td>Image Captured</td>
+                <td>
+                    10:41 am
+                    27/02/2022
+                </td>
+                <td>
+                    <button type="button" class="btn btn-primary">Go to Train</button>
+                </td>
+              </tr>
+              <tr>
+                <td>Camera #2</td>
+                <td>Face Detected</td>
+                <td>
+                    10:40 am
+                    27/02/2022
+                </td>
+                <td>
+                    <button type="button" class="btn btn-primary">Go to Train</button>
+                </td>
+              </tr>
+              <tr>
+                <td>Camera #1</td>
+                <td>Fault Detected</td>
+                <td>
+                    6:42 am
+                    27/02/2022
+                </td>
+                <td>
+                    <button type="button" class="btn btn-primary">Go to Train</button>
+                </td>
+              </tr>
```

Comment:
You can ignore all this. Just duplicated the placeholder data that was already in the table to make sure the pagination was working.